Can not compile on Msys2 (20200720)

Hello, I want to compile the last stable version (release 1.2 ) with the last Msys2 but there is some issue about that.
First it do not reconize the ./configure command.
![1](https://user-images.githubusercontent.com/67589063/89111416-c4f63580-d455-11ea-8027-cb3dc6754943.jpg)

but after ranning autoconf it shows me error on configure.ac
![2](https://user-images.githubusercontent.com/67589063/89111421-d63f4200-d455-11ea-8d3d-d531268e011b.jpg)


So !!! Any help !??

Hello this is a config.log
[config.log](https://github.com/libnet/libnet/files/5045544/config.log)

And this is a compiling log by default without changing anything.

`BSaidus@UCCEN MINGW32 ~/lnet/libnet-1.2
$ make V=1
Making all in include
make  all-am
Making all in src
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I../include  -I../include -I./../include   -O2 -Wall -I/home/BSaidus/lnet/libnet-1.2/win32/wpdpack/Include -march=i686 -mwin32 -c -o libnet_asn1.lo libnet_asn1.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../include -I../include -I./../include -O2 -Wall -I/home/BSaidus/lnet/libnet-1.2/win32/wpdpack/Include -march=i686 -mwin32 -c libnet_asn1.c  -DDLL_EXPORT -DPIC -o .libs/libnet_asn1.o
In file included from common.h:37,
                 from libnet_asn1.c:56:
C:/msys64/home/BSaidus/lnet/libnet-1.2/win32/wpdpack/Include/pcap/pcap.h:306:37: warning: 'struct bpf_program' declared inside parameter list will not be visible outside of this definition or declaration
  306 | int pcap_setfilter(pcap_t *, struct bpf_program *);
      |                                     ^~~~~~~~~~~
C:/msys64/home/BSaidus/lnet/libnet-1.2/win32/wpdpack/Include/pcap/pcap.h:316:35: warning: 'struct bpf_program' declared inside parameter list will not be visible outside of this definition or declaration
  316 | int pcap_compile(pcap_t *, struct bpf_program *, const char *, int,
      |                                   ^~~~~~~~~~~
C:/msys64/home/BSaidus/lnet/libnet-1.2/win32/wpdpack/Include/pcap/pcap.h:318:42: warning: 'struct bpf_program' declared inside parameter list will not be visible outside of this definition or declaration
  318 | int pcap_compile_nopcap(int, int, struct bpf_program *,
      |                                          ^~~~~~~~~~~
C:/msys64/home/BSaidus/lnet/libnet-1.2/win32/wpdpack/Include/pcap/pcap.h:320:27: warning: 'struct bpf_program' declared inside parameter list will not be visible outside of this definition or declaration
  320 | void pcap_freecode(struct bpf_program *);
      |                           ^~~~~~~~~~~
C:/msys64/home/BSaidus/lnet/libnet-1.2/win32/wpdpack/Include/pcap/pcap.h:321:32: warning: 'struct bpf_program' declared inside parameter list will not be visible outside of this definition or declaration
  321 | int pcap_offline_filter(struct bpf_program *, const struct pcap_pkthdr *,
      |                                ^~~~~~~~~~~
C:/msys64/home/BSaidus/lnet/libnet-1.2/win32/wpdpack/Include/pcap/pcap.h:354:31: warning: 'struct bpf_insn' declared inside parameter list will not be visible outside of this definition or declaration
  354 | u_int bpf_filter(const struct bpf_insn *, const u_char *, u_int, u_int);
      |                               ^~~~~~~~
C:/msys64/home/BSaidus/lnet/libnet-1.2/win32/wpdpack/Include/pcap/pcap.h:355:31: warning: 'struct bpf_insn' declared inside parameter list will not be visible outside of this definition or declaration
  355 | int bpf_validate(const struct bpf_insn *f, int len);
      |                               ^~~~~~~~
C:/msys64/home/BSaidus/lnet/libnet-1.2/win32/wpdpack/Include/pcap/pcap.h:356:30: warning: 'struct bpf_insn' declared inside parameter list will not be visible outside of this definition or declaration
  356 | char *bpf_image(const struct bpf_insn *, int);
      |                              ^~~~~~~~
C:/msys64/home/BSaidus/lnet/libnet-1.2/win32/wpdpack/Include/pcap/pcap.h:357:28: warning: 'struct bpf_program' declared inside parameter list will not be visible outside of this definition or declaration
  357 | void bpf_dump(const struct bpf_program *, int);
      |                            ^~~~~~~~~~~
In file included from ../include/libnet.h:107,
                 from common.h:63,
                 from libnet_asn1.c:56:
../include/./libnet/libnet-structures.h:52:5: error: unknown type name '__int64_t'
   52 |     __int64_t packets_sent;               /* packets sent */
      |     ^~~~~~~~~
../include/./libnet/libnet-structures.h:53:5: error: unknown type name '__int64_t'
   53 |     __int64_t packet_errors;              /* packets errors */
      |     ^~~~~~~~~
../include/./libnet/libnet-structures.h:54:5: error: unknown type name '__int64_t'
   54 |     __int64_t bytes_written;              /* bytes written */
      |     ^~~~~~~~~
make[1]: *** [Makefile:549: libnet_asn1.lo] Error 1
make: *** [Makefile:552: all-recursive] Error 1
`



and this is a log when change is made ( this is a portion of log not full log )
`BSaidus@UCCEN MINGW32 ~/lnet/libnet-1.2
$ make V=1 >> compile.libnet.log
In file included from common.h:37,
                 from libnet_build_802.1x.c:33:
C:/msys64/home/BSaidus/lnet/libnet-1.2/win32/wpdpack/Include/pcap/pcap.h:306:37: warning: 'struct bpf_program' declared inside parameter list will not be visible outside of this definition or declaration
  306 | int pcap_setfilter(pcap_t *, struct bpf_program *);
      |                                     ^~~~~~~~~~~
C:/msys64/home/BSaidus/lnet/libnet-1.2/win32/wpdpack/Include/pcap/pcap.h:316:35: warning: 'struct bpf_program' declared inside parameter list will not be visible outside of this definition or declaration
  316 | int pcap_compile(pcap_t *, struct bpf_program *, const char *, int,
      |                                   ^~~~~~~~~~~
C:/msys64/home/BSaidus/lnet/libnet-1.2/win32/wpdpack/Include/pcap/pcap.h:318:42: warning: 'struct bpf_program' declared inside parameter list will not be visible outside of this definition or declaration
  318 | int pcap_compile_nopcap(int, int, struct bpf_program *,
      |                                          ^~~~~~~~~~~
C:/msys64/home/BSaidus/lnet/libnet-1.2/win32/wpdpack/Include/pcap/pcap.h:320:27: warning: 'struct bpf_program' declared inside parameter list will not be visible outside of this definition or declaration
  320 | void pcap_freecode(struct bpf_program *);
      |                           ^~~~~~~~~~~
C:/msys64/home/BSaidus/lnet/libnet-1.2/win32/wpdpack/Include/pcap/pcap.h:321:32: warning: 'struct bpf_program' declared inside parameter list will not be visible outside of this definition or declaration
  321 | int pcap_offline_filter(struct bpf_program *, const struct pcap_pkthdr *,
      |                                ^~~~~~~~~~~
C:/msys64/home/BSaidus/lnet/libnet-1.2/win32/wpdpack/Include/pcap/pcap.h:354:31: warning: 'struct bpf_insn' declared inside parameter list will not be visible outside of this definition or declaration
  354 | u_int bpf_filter(const struct bpf_insn *, const u_char *, u_int, u_int);
      |                               ^~~~~~~~
C:/msys64/home/BSaidus/lnet/libnet-1.2/win32/wpdpack/Include/pcap/pcap.h:355:31: warning: 'struct bpf_insn' declared inside parameter list will not be visible outside of this definition or declaration
  355 | int bpf_validate(const struct bpf_insn *f, int len);
      |                               ^~~~~~~~
C:/msys64/home/BSaidus/lnet/libnet-1.2/win32/wpdpack/Include/pcap/pcap.h:356:30: warning: 'struct bpf_insn' declared inside parameter list will not be visible outside of this definition or declaration
  356 | char *bpf_image(const struct bpf_insn *, int);
      |                              ^~~~~~~~
C:/msys64/home/BSaidus/lnet/libnet-1.2/win32/wpdpack/Include/pcap/pcap.h:357:28: warning: 'struct bpf_program' declared inside parameter list will not be visible outside of this definition or declaration
  357 | void bpf_dump(const struct bpf_program *, int);
      |                            ^~~~~~~~~~~
In file included from common.h:37,
                 from libnet_build_802.2.c:33:
C:/msys64/home/BSaidus/lnet/libnet-1.2/win32/wpdpack/Include/pcap/pcap.h:306:37: warning: 'struct bpf_program' declared inside parameter list will not be visible outside of this definition or declaration
  306 | int pcap_setfilter(pcap_t *, struct bpf_program *);
      |                                     ^~~~~~~~~~~
....
`

--- a/README.md
+++ b/README.md
@@ -108,6 +108,47 @@ Then compile libnet with docker:
     $ ./configure
     $ make
 
+### Building with MSYS2
+
+First install all the dependencies we need including the Npcap SDK.
+
+Type in your MSYS2 terminal to install the dependencies:
+
+```bash
+pacman -S autoconf automake libtool unzip
+```
+
+Download the Npcap SDK and unzips it as a `npcap` directory to easily add it to the `configure` command:
+
+```bash
+wget https://npcap.com/dist/npcap-sdk-1.13.zip && unzip npcap-sdk-1.13.zip -d npcap
+```
+
+Download the libnet from it's [latest release][], unpack it and then generate a Makefiles:
+
+```bash
+cd libnet-x.y && ./autogen.sh
+```
+
+```bash
+CFLAGS="-I${HOME}/npcap/Include -Wno-incompatible-pointer-types" LDFLAGS="-L${HOME}/npcap/Lib/x64" ./configure --prefix=/ucrt64
+```
+
+> [!Note]
+> This is currently using a UCRT64 environment.
+
+
+Build and then install:
+
+```bash
+make && make install
+```
+
+Building Done. You can verify it's installation:
+
+```bash
+pkg-config --libs --cflags libnet
+```
 
 ### Running Unit Tests with CMocka
 

